RTN-004.tex

```diff
@@ -53,7 +57,320 @@
+This time can be distributed in any manner that is convenient to the delegate, and allocated accounts do not expire\footnote{In the case of oversubscription for the 300 RSP accounts, delegates might be asked to relinquish their accounts if they are not using them, in order to make way for others (Section \ref{ssec:res_pol}).}.
+
+In addition to the above, all DP0 delegates will be expected to:
+\begin{itemize}
+\item abide by the data rights policy in \citeds{RDO-013}
+\item adhere to security policies regarding their RSP accounts at the IDF
+\item abide by usage policies of the RSP and the IDF
+\item use the on-boarding resources to learn about the RSP
+\item follow the off-boarding steps if/when appropriate
+\item engage in DP0-related discussions via the Community Forum when possible
+\item help other DP0 delegates resolve issues when possible
+\end{itemize}
+
+Policies regarding account usage, security, and off-boarding are described in Section \ref{ssec:res_pol}, and on-boarding resources for DP0 delegates in Section \ref{ssec:res_sup}. 
+
+
+\section{Delegate Selection Process}\label{sec:sel}
+
+There will be 300 RSP user accounts at the IDF available for science community DP0 delegates, in addition to the ~150 accounts for Rubin staff members.
+This marks an almost order of magnitude increase in the number of community accounts in the current NCSA instance of the RSP (via the Stack Club).
+
+The number of RSP accounts remains constrained during the Rubin Observatory construction and pre-operations phase -- and especially for DP0 -- due to the Rubin pre-operations team's limited scope in providing support for data software and services that are still in development.
+Time is needed to scale up in a sustainable manner.
+At least by the start of Rubin Observatory Operations, all data rights holders will be able to have RSP accounts.
+It is a primary goal of the Rubin pre-operations team to provide RSP access as soon as possible to all data rights holders. 
+
+All DP0 delegates must be data rights holders, e.g., US or Chilean scientists, or individuals associated with an International Contribution (in-kind) proposal in progress \citeds{RDO-013}.
+
+\subsection{Delegate Groups}\label{ssec:sel_grps}
+
+Achieving the goals of DP0 requires a broad diversity in the DP0 delegates, each with their own experience, expertise, or novice perspective, and each representing and reporting back to different parts of the broader community.
+
+The primary goal of this application process is to ensure that the final set of DP0 delegates includes the full range of diversity in the science community, and provides the beneficial opportunities of DP0 participation equitably to them.
+
+To guide the selection process, the groups of DP0 delegates are defined in Table \ref{tab:delegate_groups}, and the \emph{approximate} number of delegates in each group estimated.
```

Comment:
I re-iterate my request to allow the stack club in this allocation. They are very high value early users because (1) they can provide us with direct feedback on the comparison between the IDF and LDF user experience and (2) they have shown sustained commitment to working with early versions of our products

Reply:
Stack Club members would primarily be in Group B, "experienced users of science platforms (e.g., RSP, SciServer, DataLab)" -- they're the ones with RSP experience. To make an explicit mention of Stack Club I added this sentence to the end of Section 2.1 "Delegate Groups":

For example, Stack Club\footenote{\url{https://github.com/LSSTScienceCollaborations/StackClub#stack-club}} members might self-identify with Group B due to their RSP experience, but could also self-identify with any other group(s).